src/main/java/kb/weather/location/Location.java

```diff
@@ -0,0 +1,25 @@
+package kb.weather.location;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.util.UUID;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Location {
+    @Id
+    private String uuid = UUID.randomUUID().toString();
```

Comment:
Use uuid in @GeneratedValue and @GenericGenerator

Reply:
Ok, done.